### PR TITLE
Add aws-ebs-csi-driver-operator jobs

### PR DESCRIPTION
**Pull request by jsafrane**
Starting with image and unit tests.

### CODE REVIEW

**Comment by jsafrane** [comment]
Finally green & ready for review.

File: ci-operator/config/openshift/aws-ebs-csi-driver-operator/openshift-aws-ebs-csi-driver-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
+      paths: null
+  to: aws-ebs-csi-driver-operator
+promotion:
+  name: "4.5"
+  namespace: ocp
+resources:
+  '*':
+    limits: {}
```
Comment (droslean) [inline ci-operator/config/openshift/aws-ebs-csi-driver-operator/openshift-aws-ebs-csi-driver-operator-master.yaml]: no need to put an empty value here.

Comment (jsafrane) [reply ci-operator/config/openshift/aws-ebs-csi-driver-operator/openshift-aws-ebs-csi-driver-operator-master.yaml]: Fixed

**Pull request merged**